lib/pliny/metrics.rb

```diff
@@ -0,0 +1,20 @@
+module Pliny
+  module Metrics
+    def count(*names, value: 1)
+      counts = Hash[names.map { |n| ["count##{Config.app_name}.#{n}", value] }]
+      Pliny.log(counts)
+    end
+
+    def measure(*names, &block)
+      start = Time.now
+      return_value = block.call
+      elapsed = Time.now - start
+
+      measures = Hash[names.map { |n| ["measure##{Config.app_name}.#{n}", elapsed] }]
```

Comment:
This is slightly obscure, and the exact same thing as in `#count`. How about extracting it into another method? Something like:

```
def monitor(key, names, value)
  data = Hash[names.map { |n| ["#{key}##{Config.app_name}.#{n}", value] }]
  Pliny.log(data)
end
```